public/src/main.tsx

```diff
@@ -144,111 +151,109 @@ const AppRouter = () => {
+          path="/header-tests/:testName?"
+          render={(): React.ReactElement => createComponent(<HeaderTestsForm />, 'Header Tests')}
+        />
+        <Route
+          path="/epic-tests/:testName?"
+          render={(): React.ReactElement => createComponent(<ArticleEpicTestsForm />, 'Epic Tests')}
+        />
+        <Route
+          path="/liveblog-epic-tests/:testName?"
+          render={(): React.ReactElement =>
+            createComponent(<LiveblogEpicTestsForm />, 'Liveblog Epic Tests')
+          }
+        />
+        <Route
+          path="/apple-news-epic-tests/:testName?"
+          render={(): React.ReactElement =>
+            createComponent(<AppleNewsEpicTestsForm />, 'Apple News Epics')
+          }
+        />
+        <Route
+          path="/amp-epic-tests/:testName?"
+          render={(): React.ReactElement => createComponent(<AMPEpicTestsForm />, 'AMP Epics')}
+        />
+        <Route
+          path="/banner-tests/:testName?"
+          render={(): React.ReactElement => createComponent(<BannerTestsForm1 />, 'Banner Tests 1')}
+        />
+        <Route
+          path="/banner-tests2/:testName?"
+          render={(): React.ReactElement => createComponent(<BannerTestsForm2 />, 'Banner Tests 2')}
+        />
+        <Route
+          path="/banner-deploy"
+          render={(): React.ReactElement =>
+            createComponent(<BannerDeployDashboard />, 'Banner Deploy')
+          }
+        />
+        <Route
+          path="/channel-switches"
+          render={(): React.ReactElement =>
+            createComponent(<ChannelSwitches />, 'Channel Switches')
+          }
+        />
+        <Route
+          path="/campaigns/:campaignName?"
+          render={(): React.ReactElement => createComponent(<CampaignsForm />, 'Campaigns')}
+        />
+        <Route
+          path="/qr-code"
+          render={(): React.ReactElement => createComponent(<QrCodePage />, 'QR Code Generator')}
+        />
+        <Route
+          path="/apps-metering-switches"
+          render={(): React.ReactElement =>
+            createComponent(<AppsMeteringSwitches />, 'Apps Metering Switches')
+          }
+        />
+        <Route
+          path="/super-mode"
+          render={(): React.ReactElement =>
+            createComponent(<SuperModeDashboard />, 'Epic Super Mode dashboard 🦸')
+          }
+        />
+        <Route
+          path="/default-promos"
+          render={(): React.ReactElement => createComponent(<DefaultPromos />, 'Default Promos')}
+        />
+        <Route
+          path="/banner-designs/:name?"
+          render={(): React.ReactElement => createComponent(<BannerDesigns />, 'Banner Designs')}
+        />
+      </div>
+    </Router>
   );
 };
 
 const container = document.getElementById('root');
 if (container) {
   const root = createRoot(container);
-  root.render(<AppRouter />);
+  root.render(
+    <ThemeProvider theme={getTheme()}>
+      <StyledEngineProvider injectFirst>
```

Comment:
this prevents our styles from being overridden:
https://mui.com/material-ui/migration/v5-style-changes/#%E2%9C%85-adjust-css-injection-order